test/distributed/org/apache/cassandra/distributed/test/thresholds/TombstoneCountWarningTest.java

```diff
@@ -248,7 +274,10 @@ public boolean matches(InetAddress value)
 
         assertWarnAborts(0, 2, 1);
 
+        // when disabled warnings only happen if on the coordinator, and coordinator may not be the one replying
+        // to every query
         enable(false);
+        State.blockFor(CLUSTER.get(1).config().broadcastAddress());
```

Comment:
honestly `driverQueryAll(cql)` below could use the same, but for some reason its stable where as this wasn't...